### ISSUE TITLE
EXT-JWT authentication can return a 401, which is out of spec

### ISSUE THREAD

**Issue by andrewpmartinez** [openziti/ziti#1591]
*Experienced*

If JWT based authentication fails, a 401 is returned. Which is not defined in the OAS 2.0 spec and causes generated clients to return non-standard errors.

*Expected*

Document the 401 in the OAS2.0 specification.

*Reproduction*

1) stand up a network
2) add an external JWT signer
3) configure an authentication policy to allow JWT authentication and allow the newly created signer
4) attempt to authenticate with an invalid JWT

```
level=ERROR ts=2023-04-11T17:01:27.021902706Z app=core-command source=httpserver.go:185 msg="could not authenticate: response status code does not match any response statuses defined for this endpoint in the swagger spec (status 401): {}"
```

**Comment by smilindave26** [comment]
Andrew to verify if already fixed, and if not will add to the IdP project board

**Comment by andrewpmartinez** [comment]
This was fixed in commit #65a5f7da83b3c1d561e2233babc38a91ee3e469b before repo merging.

**Comment by andrewpmartinez** [comment]
This is currently fixer per this line: https://github.com/openziti/edge-api/blob/01cf419458a5bd7e3724b74927113227fceb3877/source/shared/authenticate.yml#L36
